### PR TITLE
Alex/landing/static blocks

### PR DESCRIPTION
**Pull request by AlexAzalor**

### CODE REVIEW

File: src/components/home/advantages.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import { useTranslations } from "next-intl";
+import Image from "next/image";
+
+const advantages = [
```
Comment (Taraslysak) [inline src/components/home/advantages.tsx]: Please, use CAPS for constants

File: src/components/layout/footer.tsx [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+  const goToHome = () => {
+    router.push("/");
+  };
+
+  return (
+    <div className="min-h-[496px] bg-blackMain">
+      <div className="p-20 desktopEnd:px-4">
+        <div className="flex justify-between desktopEnd:flex-col desktopEnd:items-center desktopEnd:gap-10 desktopEnd:text-center">
+          <div>
+            <div
```
Comment (Taraslysak) [inline src/components/layout/footer.tsx]: May be use `Link` from next?

File: tailwind.config.ts [hunk 3]
```diff
@@ -16,6 +16,7 @@ const config = {
       desktopEnd: { max: "672px" },
       mdDesktop: { max: "1366px" },
       smDesktop: { max: "1184px" },
+      smDesktop2: { min: "673px", max: "1184px" },
```
Comment (Taraslysak) [inline tailwind.config.ts]: why do we need this? won't it conflict with desktopEnd and smDesktop???

**Review by Taraslysak — COMMENTED** [review]
Please consider some small refactorings

File: tailwind.config.ts (hunk 3 above)
Comment (AlexAzalor) [reply tailwind.config.ts]: No, this is necessary so that the styles will change correctly when changing screens.

**Pull request merged**